### ISSUE TITLE
Dropdowns are missing screenreader labels

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4861]
The dropdowns for “creator”, “Collection” etc. are read out loud using a screen reader and what is inside of the field, but it does not say that the input is for the creator field or the collection field.

The form elements have missing labels. People who use a screen reader need a label to hear what the element is for. Placeholder text is not allowed as label.

**Comment by Floppy** [comment]
I think this has been already dealt with as part of the form tidying up / component conversion
